tools/ldb_cmd.cc

```diff
@@ -114,11 +114,83 @@ const std::string LDBCommand::ARG_READ_TIMESTAMP = "read_timestamp";
 const char* LDBCommand::DELIM = " ==> ";
 
 namespace {
+// Helper class to iterate WAL logs in a directory in chronological order.
+class WALPicker {
```

Comment:
Minor but how about calling this something like `WALFileIterator` ? `WALPicker` suggests to me it will pick a single one.

Reply:
Thanks for the suggestion, `WALFileIterator` is more clear.